### ISSUE TITLE
Password logged in plain in version 1.3.1

### ISSUE THREAD

**Issue by chrissng** [neo4j/neo4j-python-driver#173]
Neo4j driver session is created using basic auth:
``` python
@contextmanager
def get_neo4j_context(uri):
    """Get the Neo4J context manager"""
    neo4j_context = None
    try:
        decoded = decode_url_connection_string(uri)
        auth = basic_auth(decoded['username'], decoded['password'])
        driver = GraphDatabase.driver(uri, auth=auth)
        neo4j_context = driver.session()
        yield neo4j_context
    except (AddressError, ProtocolError,
            ServiceUnavailable, SecurityError) as graph_exception:
        raise graph_exception
    finally:
        if neo4j_context:
            neo4j_context.close()
            neo4j_context = None
```

Neo4j logs with the password shown in plain:
```bash
INFO - ~~ [CONNECT] ('192.168.150.46', 7687)
INFO - ~~ [SECURE] 192.168.150.46
INFO - C: [HANDSHAKE] 0x6060B017 [1, 0, 0, 0]
INFO - S: [HANDSHAKE] 1
INFO - C: INIT (u'neo4j-python/1.3.1', {'credentials': u'ovlB5lft&$', 'scheme': 'basic', 'principal': u'neo4j'})
INFO - S: SUCCESS ({u'server': u'Neo4j/3.1.3'})
```

Is there a means to not log credentials in INFO, or allow the client to set logging level to something else?

**Comment by chrissng** [comment]
My current solution to address this:

``` python
@contextmanager
def get_neo4j_context(uri):
    """Get the Neo4J context manager"""
    neo4j_context = None
    try:
        # Set neo4j bolt logging level to WARNING
        log = logging.getLogger("neo4j.bolt")
        log.setLevel(logging.WARNING)

        ...
```

**Comment by chrissng** [comment]
Understand that this is fixed in 1.4.0b1: https://github.com/neo4j/neo4j-python-driver/commit/50ad7192246a8c12dd277bc8ca60d86fa7e07345

I am using neo4j-driver version 1.3.1

**Comment by technige** [comment]
This is fixed in https://github.com/neo4j/neo4j-python-driver/commit/50ad7192246a8c12dd277bc8ca60d86fa7e07345
